submissions/hadizaSaadu/HTML/E-commerce.html

```diff
@@ -13,7 +13,10 @@ <h3>
     <a href="#">HOME</a>
 
     <a href="#">ABOUT US</a>
-    <a href="#">CONTACT</a>
+    <a
+      href="file:///C:/Users/hp/Desktop/Dufuna-CodeCamp20/submissions/hadizaSaadu/html-forms/contact-page.html?sname=&email=&subject="
```

Comment:
This path won't work in another user computer if your work is clone cos this isn't relative. Same as below

Reply:
okay ...done and repushed...thanks